api_messages.go

```diff
@@ -2,9 +2,29 @@ package sockchat
 
 import (
 	"encoding/json"
+	"fmt"
 	"log"
 )
 
+const (
+	LoginAction       = "login"
+	JoinAction        = "join"
+	CreateAction      = "create"
+	LeaveAction       = "leave"
+	SendMessageAction = "send_message"
+
+	UserJoinedChannelEvent = "user has joined the channel"
+	UserLeftChannelEvent   = "user has left the channel"
+	YouLeftChannelEvent    = "you have left the channel"
+	NewMessageEvent        = "new message in channel"
```

Comment:
This constant is unused

Reply:
fixed it, it's now used in tests & user_connections

Reply:
> btw which tool did you use for this kind of check?

I've went through all of your files one by one and my IDE highlighted it

Reply:
Also JetBrains Goland has Code Inspection tool which shows many more potential issues:
![Screenshot 2023-04-18 at 15 17 03](https://user-images.githubusercontent.com/28757872/232789491-f004450c-435a-4daf-89be-00ce3bcba9ae.png)